build/Packages.props

```diff
@@ -0,0 +1,59 @@
+<Project>
+
+  <PropertyGroup>
+      <NuGetPackageVersion>4.8.0-preview1.5185</NuGetPackageVersion>
+
+      <!-- Prodcon-overrideable, using repo-toolset naming convention -->
+      <NuGetBuildTasksVersion>$(NuGetPackageVersion)</NuGetBuildTasksVersion>
+      <NuGetCommandsVersion>$(NuGetPackageVersion)</NuGetCommandsVersion>
+      <NuGetProtocolVersion>$(NuGetPackageVersion)</NuGetProtocolVersion>
```

Comment:
FYI this is where I would expect to add any dependencies from upstream providers to show up. So if ProdCon specifies a version it would be picked up here and specified below using that property. Should work exactly as it did before.

Reply:
Oh, I see. I get it now.

Reply:
Should be good then.